server/generate.py

```diff
@@ -292,8 +292,11 @@ def seed_scores():
     admin = User.query.filter_by(is_admin=True).first()
     for backup in Backup.query.filter_by(submit=True).all():
         if random.choice([True, False]):
-            score = gen_score(backup, admin)
-            db.session.add(score)
+             score = gen_score(backup, admin, kind='composition')
+             db.session.add(score)
+        if random.choice([True, False]):
+             score = gen_score(backup, admin, kind='Total')
```

Comment:
I think that's how the 61A autograder has been sending grades back to OK - I'll lowercase this for now. 

Side note: Ok-client shouldn't use Total because it's confusing to students - Total could imply it includes the composition score.